packages/data-vault-client/src/authorization/zcap.sanity.test.ts

```diff
@@ -0,0 +1,38 @@
+import { signCapabilityInvocation } from 'http-signature-zcap-invoke';
+import { CapabilityInvoker } from '../CapabilityInvoker';
+import * as ed25519 from '@transmute/did-key-ed25519';
+
+import base64url from 'base64url';
```

Comment:
Should this be:

```suggestion
import base64url from 'base64url-universal';
```

To match the package.json file? If so, another line below should change...